### ISSUE TITLE
Add description field to BlogPage model

### ISSUE THREAD

**Issue by Rascat** [bocian67/dezentrale_web#49]
A description can be useful piece of information that introduces the reader to the topic that is dealt with in the text.
We could display the description in the blog index page

**Comment by bocian67** [comment]
The introduction is simply not able to be displayed in the index page - but why?

**Comment by elbenfreund** [comment]
I do not understand you last comment @bocian67 . Please note that this is the "issue" discussion, which should be different from discussing a particular PR ;)

With regards to the issue: I think ``intro`` or ``teaser`` would be a more common name for what you seem to have in mind. what are your thoughts?

**Comment by bocian67** [comment]
Isn't it similar to #50?

**Comment by elbenfreund** [comment]
That depends. Is this issue about *adding a field to the model* or is it about *displaying a field that is already present*?

**Comment by Rascat** [comment]
It is similar, but not the same. This issue is about _adding a field to the mode_ . Issue #50 is about displaying that information in the template

**Comment by elbenfreund** [comment]
@Rascat that was exacly my point :)

**Comment by elbenfreund** [comment]
Can we please answer my original question, though?

>With regards to the issue: I think ``intro`` or ``teaser`` would be a more common name for what you seem to have in mind. what are your thoughts? Such an attribute already [exists](https://github.com/bocian67/dezentrale_web/blob/develop/dezentrale_web/apps/blog/models.py#L9). If so, this issue seems to be a ``wont't fix`` and can be closed.

**Comment by bocian67** [comment]
yes and no. I understand your confusion, but the Intro which you mean is kind of an introduction text which purpose is to introduce visitors on the blogindex page. The other description (which is an intro too) is specific for each blog entry, and introduces to the blog to decide if you want to read it or not. So both are intro-texts but for another purpose.

**Comment by elbenfreund** [comment]
Oh. you are right of cause. My bad! :( Maybe it would be good to be consistent with the naming though and call it ``BlogPage.intro`` to?

**Comment by bocian67** [comment]
Is this still relevant or can it be closed?

**Comment by elbenfreund** [comment]
well, you and @Rascat brought it up, so it seems like you are the ones to anwer your question... ;)

**Comment by bocian67** [comment]
Yes but I'm sure if we already solved it in develop or latest PR.

Am 30.11.2017 11:37 schrieb "Eric Goller" <notifications@github.com>:

**Comment by elbenfreund** [comment]
well, there is an easy way to find out... :)

**Comment by bocian67** [comment]
I changed the model name to clarify it, but tox throws an error (tox_pyenv.PyenvMissing), but I couldn't figure out why tox wants to find pyenv, since we don't use it, and even updating the packages locally to the newest version couln't fix it. @elbenfreund do you have experienced that before?
The Issue itself should be fixed.
